test/exec_mode_float_control_khr.ll

```diff
@@ -0,0 +1,66 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv --spirv-ext=+SPV_KHR_float_controls
+; RUN: llvm-spirv %t.spv -o %t.spt --to-text
+; RUN: FileCheck %s --input-file %t.spt -check-prefix=SPV
```

Comment:
Please launch validator ([1](https://github.com/KhronosGroup/SPIRV-Tools#validator), [2](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/180)) on the produced SPIR-V: if I understand correctly, right now this test won't produce valid SPIR-V, because of the presence of several execution modes on the same kernel


UPD: if I understand correctly, the validator doesn't catch such errors right now, but it will be fixed at some point, I guess: KhronosGroup/SPIRV-Tools#2150

Reply:
Do you say that there could be only one execution mode for kernel?
I cannot find that in spec...

Reply:
Fixed